main.py

```diff
@@ -141,17 +205,27 @@ def restore_window():
 ctk.set_default_color_theme("blue")  # Themes: "blue" / "green" / "dark-blue"
 
 panel = ctk.CTk()
-panel.geometry("400x270+"+ str(int(screen_width/2) - 200) + '+' + str(int(screen_height/2) - 200))
+panel.geometry("400x300+"+ str(int(screen_width/2) - 200) + '+' + str(int(screen_height/2) - 200))
 panel.resizable(False, False)
 panel.title('NoMoreBorder')
 
 
-label = ctk.CTkLabel(panel, text="Display Resolution is " + str(screen_width) + 'x' + str(screen_height), font = ("Helvetica", 20))
+label = ctk.CTkLabel(panel, text="Display Resolution is " + str(monitors[selected_monitor].width) + 'x' + str(monitors[selected_monitor].height), font = ("Helvetica", 20))
```

Comment:
Since this is being managed by selected monitor's dimensions. I think SetProcessDPIAware() [Line 11] was bugging it out and reported my 2560x1600 at 200% scale as 1280x800. I will disable that on main once merged.